Monika After Story/game/script-story-events.rpy

```diff
@@ -516,6 +516,7 @@ label mas_player_name_enter_name_loop(input_prompt):
             ).strip(' \t\n\r')
 
             lowername = tempname.lower()
+            alphaname = re.sub("[^a-zA-Z0-9]", "", tempname)
```

Comment:
I would suggest using `\s` as that should catch any kind of space while allowing every other symbol (think about translations that might allow different characters, or maybe if we get proper unicode support in the future).